flagsmith-core.js

```diff
@@ -169,6 +170,23 @@ const Flagsmith = class {
                 throw ('Please specify a environment id');
             }
 
+            if (this.sendFlagEvaluationEvents) {
+                this.interval.push(setInterval(this.analyticsFlags, this.ticks))
```

Comment:
Interval shouldn't be an array, there should never be more than 1 set interval to post analytics. 

Also, this needs to check whether an interval was already initialised.